docs/source/daml/reference/updates.rst

```diff
@@ -87,10 +101,37 @@ fetch
 
    fetchedContract <- fetch IdOfContract
 
-- ``fetch`` keyword.
+- ``fetch`` function.
 - Fetches the contract instance with that ID. Usually used with a bound variable, as in the example above.
 - Often used to check the details of a contract before exercising a choice on that contract. Also used when referring to some reference data.
-- ``fetch cid`` fails if ``cid`` is not the contract id of an active contract, and thus causes the entire transaction to abort. 
+- ``fetch cid`` fails if ``cid`` is not the contract id of an active contract, and thus causes the entire transaction to abort.
+- The submitting party must be an observer or signatory on the contract, otherwise ``fetch`` fails, and similarly causes the entire transaction to abort.
+
+.. _daml-ref-fetch-by-key:
+
+fetchByKey
+**********
+
+.. code-block:: daml
+
+   fetchedContract <- fetchByKey @ContractType contractKey
+
+- ``fetchByKey`` function.
+- The same as ``fetch``, but fetches the contract instance with that :doc:`contract key </daml/reference/contract-keys>`, instead of the contract ID.
```

Comment:
Shouldn't we say that we require authorization from at least one of the key maintainers?